buildSrc/src/main/java/Dependencies.kt

```diff
@@ -29,21 +29,21 @@ object Dependencies {
 
         private const val lifecycleVersion = "2.3.0-rc01"
 
-        const val activity_ktx = "androidx.activity:activity-ktx:1.2.0-rc01"
+        const val activity_ktx = "androidx.activity:activity-ktx:1.3.0-alpha03"
         const val annotation = "androidx.annotation:annotation:1.1.0"
         const val autofill = "androidx.autofill:autofill:1.1.0"
         const val appcompat = "androidx.appcompat:appcompat:1.3.0-beta01"
-        const val biometric_ktx = "androidx.biometric:biometric-ktx:1.2.0-alpha02"
+        const val biometric_ktx = "androidx.biometric:biometric-ktx:1.2.0-alpha03"
         const val constraint_layout = "androidx.constraintlayout:constraintlayout:2.1.0-alpha2"
-        const val core_ktx = "androidx.core:core-ktx:1.5.0-beta01"
+        const val core_ktx = "androidx.core:core-ktx:1.5.0-beta02"
         const val documentfile = "androidx.documentfile:documentfile:1.0.1"
         const val fragment_ktx = "androidx.fragment:fragment-ktx:1.3.0-rc02"
```

Comment:
```suggestion
        const val fragment_ktx = "androidx.fragment:fragment-ktx:1.3.0"
```

Apparently I missed this release.